pkg/rpc/response/result/asset_state.go

```diff
@@ -20,7 +20,7 @@ type AssetState struct {
 	Admin      string                `json:"admin"`
 	Issuer     string                `json:"issuer"`
 	Expiration uint32                `json:"expiration"`
-	IsFrozen   bool                  `json:"is_frozen"`
+	IsFrozen   bool                  `json:"frozen"`
```

Comment:
There is also some custom logic with `Name` field, which we are missing:
https://github.com/neo-project/neo/blob/master-2.x/neo/Ledger/AssetState.cs#L156
https://github.com/neo-project/neo/blob/master-2.x/neo/Ledger/AssetState.cs#L95

Reply:
I see this is mentioned later in implementation details.